### ISSUE TITLE
Clear chat options starts a new chat

### ISSUE THREAD

**Issue by kliu57** [tarasglek/chatcraft.org#305]
I noticed that the Clear chat option starts a new chat rather than clearing the current chat. Is this a bug?

![clearChat1](https://github.com/tarasglek/chatcraft.org/assets/98062538/cea5f1db-f942-48a7-9298-beabdcf95d2e)

![clearChat2](https://github.com/tarasglek/chatcraft.org/assets/98062538/e4a3e7cd-5eff-43c3-81d9-ddbb4ef814e8)

**Comment by humphd** [comment]
We've gone through various iterations of this.  The difference between "Clear Chat" and "New Chat" is really about what happens in this current tab.  As you point out, both create a new chat, but "Clear Chat" clears the current tab "New Chat" does it in a new window/tab.

We used to do what you're suggesting, and simply delete the messages in the window.  However, users complained about data loss (it was surprising that they lost their messages), so we switched to this.

In short, we err on the side of keeping vs. deleting data.

I don't think we want to change this, since it took a lot to get here.

**Comment by kliu57** [comment]
Closing issue as it is expected behaviour
